cmake/AddEventLibrary.cmake

```diff
@@ -117,7 +150,13 @@ macro(add_event_library LIB_NAME)
         PUBLIC_HEADER DESTINATION "include/event2"
         COMPONENT dev
     )
-
+    if (SOLARIS OR ${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
can you instead do something like `if (not windows)`

(seems like FreeBSD and OpenBSD would not be covered by this condition?)

Reply:
Looks like the token is `WIN32` so I think this should just be `if (NOT WIN32)`